src/avro.erl

```diff
@@ -167,6 +168,10 @@ decode_schema(JSON, Options) ->
 encode_schema(Type) ->
   avro_json_encoder:encode_schema(Type).
 
+-spec encode_schema(avro_type(), map()) -> binary().
```

Comment:
reuse `schema_ops()` i.e. proplist

Reply:
Changed to schema_opts() for public interface.